packages/web/components/Modal/ShareModal/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback, useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import { toast } from '../../Toast';
+import {
+	Container,
+	IconsWrapper,
+	IconButton,
+	CopyIcon,
+	LocationInput,
+	FacebookIcon,
+	TwitterIcon,
+	LinkedinIcon,
+	WhatsAppIcon,
+} from './styles';
+
+const socialNetworks = {
+	facebook: {
+		url: 'https://facebook.com/sharer/sharer.php?u=:url',
+		icon: FacebookIcon,
+	},
+	twitter: {
+		url: `https://twitter.com/intent/tweet?url=:url`,
+		icon: TwitterIcon,
+	},
+	linkedin: {
+		url: 'https://www.linkedin.com/sharing/share-offsite/?url=:url',
+		icon: LinkedinIcon,
+	},
+	whatsapp: {
+		url: 'https://api.whatsapp.com/send?text=:url',
+		icon: WhatsAppIcon,
+	},
+};
+
+const ShareModal = () => {
+	const { t } = useTranslation(['common']);
+	const pageUrlRef = useRef(null);
+
+	const handleSocialShare = useCallback((service) => {
+		const pageUrl = encodeURIComponent(pageUrlRef.current.value);
+		const serviceUrl = socialNetworks[service].url.replace(':url', pageUrl);
+		window.open(serviceUrl, '_blank');
```

Comment:
é interessante definir o `opener` como null para evitar problemas de segurança.

https://dev.to/dhilipkmr/why-should-you-use-noopener-beware-of-security-flaws-3i57